doc/design/cluster_train/data_dispatch.md

```diff
@@ -92,11 +92,11 @@ random_images-00099-of-00099
 
 #### 进行训练
 
-PaddlePaddle提供专用的[data reader creator](https://github.com/PaddlePaddle/Paddle/blob/develop/doc/design/reader/README.md#python-data-reader-design-doc)，生成给定SSTable文件对应的data reader。**无论在本地还是在云端，reader的使用方式都是一致的**：
+PaddlePaddle提供专用的[data reader creator](https://github.com/PaddlePaddle/Paddle/blob/develop/doc/design/reader/README.md#python-data-reader-design-doc)，生成给定RecordIO文件对应的data reader。**无论在本地还是在云端，reader的使用方式都是一致的**：
 
 ```python
 # ...
-reader = paddle.reader.creator.SSTable("/home/random_images-*-of-*")
+reader = paddle.reader.creator.RecordIO("/home/random_images-*-of-*")
```

Comment:
I think `pfs://` only have to be a concept for the fileserver client to distinguish between remote and local.
Since we always mount the user home directory on the same place, users can just think it as local directory. I think there is no need to add the concept of `pfs` for user to understand.